recipes/glm/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "GLM" %}
+{% set version = "0.9.9.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/g-truc/glm/releases/download/{{ version }}/glm-{{ version }}.zip
+  sha256: 209b5943d393925e1a6ecb6734e7507b8f6add25e72a605b25d0d0d382e64fd4
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
```

Comment:
I've been going around adding `- make   # [unix]` as a build dependency for many packages. Not really important now though.